packages/presentation/lib/screen/login/login_data.dart

```diff
@@ -1,13 +1,23 @@
 class LoginData {
-  const LoginData({required this.errorMessage});
+  const LoginData({
+    required this.errorMessage,
+    this.validateLogin,
+    this.validatePassword,
+  });
 
   final String errorMessage;
```

Comment:
if I'm not mistaken, you don't use this variable anywhere?

Reply:
no, I use it in bloc
But I find it unnecessary. Removed it